src/neptune_tensorboard/sync/sync_impl.py

```diff
@@ -9,6 +9,7 @@
 try:
     import tbparse
 except ModuleNotFoundError:
+    # user facing
     raise ModuleNotFoundError("neptune-tensorboard: require `tbparse` for exporting logs (pip install tbparse)")
```

Comment:
Maybe the phrasing could be slightly clearer here, if I understand it correctly.

```suggestion
    raise ModuleNotFoundError("neptune-tensorboard: tbparse is required for exporting logs. Install it with: pip install tbparse")
```